python-ffi/Data/Array.py

```diff
@@ -6,15 +6,15 @@
 def _rangeImpl(start):
     def ap(end):
         step = 1 if start <= end else -1
-        return list(_buitins["range"](start, end + step, step))
+        return tuple(_buitins["range"](start, end + step, step))
```

Comment:
`_builtins["range"]` is very slow. please directly load `range` instead. You can give an alias to range in global context, like
`_glob = global`.